### PR TITLE
Add example for AFTER crud operations option

### LINKED ISSUE #454: Support "after" CRUD operation option

### PR DESCRIPTION
**Pull request by nickkkccc**
I haven't forgotten about:
- [ ] Tests
- [ ] Changelog
- [ ] Documentation
- [ ] Commit messages comply with the [guideline](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message)
- [ ] Cleanup the code for review. See [checklist](https://github.com/tarantool/cartridge-java/blob/master/docs/review-checklist.md)

Related issues:
Closes #454

### CODE REVIEW

File: src/test/java/io/tarantool/driver/integration/proxy/options/ProxySpaceSelectOptionsIT.java [hunk 1]
```diff
@@ -174,4 +174,38 @@ public void withModeTest() throws ExecutionException, InterruptedException {
...
+            assertEquals(insertedTuple.getObject(0), tuple.getObject(0));
+            assertEquals(insertedTuple.getObject(2), tuple.getObject(2));
+            assertEquals(insertedTuple.getObject(3), tuple.getObject(3));
+            assertEquals(insertedTuple.getObject(4), tuple.getObject(4));
+        }
+
+        TarantoolResult<TarantoolTuple> resultAfterInsert = profileSpace.select(Conditions.any()).join();
+        assertEquals(tupleCount, resultAfterInsert.size());
+
+        final int halfOfTupleCount = tupleCount / 2;
```
Comment (ArtDu) [inline src/test/java/io/tarantool/driver/integration/proxy/options/ProxySpaceSelectOptionsIT.java:201]:
```suggestion
        final int median = tupleCount / 2;
```

File: src/test/java/io/tarantool/driver/integration/proxy/options/ProxySpaceSelectOptionsIT.java [hunk 2]
```diff
@@ -174,4 +174,38 @@ public void withModeTest() throws ExecutionException, InterruptedException {
...
+        TarantoolTuple afterTuple = tupleFactory.create(halfOfTupleCount,
```
Comment (ArtDu) [inline src/test/java/io/tarantool/driver/integration/proxy/options/ProxySpaceSelectOptionsIT.java:202]:
```suggestion
        TarantoolTuple medianTuple = tupleFactory.create(halfOfTupleCount,
```

**Review by ArtDu — COMMENTED** [review]
LGTM, только halfOfTupleCount звучит сложно, медиана попроще

File: src/test/java/io/tarantool/driver/integration/proxy/options/ProxySpaceSelectOptionsIT.java (hunk 1 above)
Comment (nickkkccc) [reply src/test/java/io/tarantool/driver/integration/proxy/options/ProxySpaceSelectOptionsIT.java]: fixed

File: src/test/java/io/tarantool/driver/integration/proxy/options/ProxySpaceSelectOptionsIT.java (hunk 2 above)
Comment (nickkkccc) [reply src/test/java/io/tarantool/driver/integration/proxy/options/ProxySpaceSelectOptionsIT.java]: fixed

**Review by ArtDu — APPROVED** [review]

**Review by ArtDu — APPROVED** [review]

**Pull request merged**